plexapi/base.py

```diff
@@ -321,17 +464,24 @@ def findItems(self, data, cls=None, initpath=None, rtag=None, **kwargs):
             kwargs['type'] = cls.TYPE
         # rtag to iter on a specific root tag using breadth-first search
         if rtag:
-            data = next(utils.iterXMLBFS(data, rtag), [])
+            data = next(utils.iterXMLBFS(data, rtag), Element('empty'))
         # loop through all data elements to find matches
-        items = []
+        items: List[Union[PlexObject, PlexObjectT]] = []
```

Comment:
```suggestion
        items: list[Union[PlexObject, PlexObjectT]] = []
```

afaik can use native list

Reply:
Not in Python 3.8.

Reply:
eeh. why? any reference?

as it works in my project:
- https://github.com/Taxel/PlexTraktSync/blob/598685fe5fc4f3ae7acd7927218f5f9bc7a71352/setup.cfg#L20
- https://github.com/Taxel/PlexTraktSync/blob/598685fe5fc4f3ae7acd7927218f5f9bc7a71352/plextraktsync/util/execx.py#L6

pyupgrade config:
- https://github.com/Taxel/PlexTraktSync/blob/598685fe5fc4f3ae7acd7927218f5f9bc7a71352/.pre-commit-config.yaml#L44-L49

Reply:
You need `from __future__ import annotations`.

Reply:
I think what happens here is that if you import `__annotations__`, you can use `list[]`:
- https://stackoverflow.com/questions/39458193/using-list-tuple-etc-from-typing-vs-directly-referring-type-as-list-tuple-etc/62033243#62033243

unless it's also a generic:
- https://stackoverflow.com/questions/39458193/using-list-tuple-etc-from-typing-vs-directly-referring-type-as-list-tuple-etc/39458225#39458225